### PR TITLE
Add Getting Started Doc

### PR DESCRIPTION
**Pull request by IAmHughes**
This begins adding the `Getting Started` doc to the docs repo.

@khorne3 I've not added it to the manifest as I wasn't sure where you wanted it to live. Would love an initial review and we can iterate from here!

### CODE REVIEW

**Comment by khorne3** [comment]
We need an icon, but not too shabby:

Screen Shot 2021-07-02 at 2 55 17 PM

**Comment by ericpaulsen** [comment]
where do we source our icons from? i'll keep an eye out any potential candidates.

**Comment by IAmHughes** [comment]
> We need an icon, but not too shabby:

I vote for a 🚀 or ☕ or something.

> where do we source our icons from?

I don't know specifically where, but these are material design I believe and [a site like this](https://material.io/design/iconography/system-icons.html#design-principles) would have a load of them.

File: getting-started.md
Comment (bpmct) [inline getting-started.md:15]: I would suggest unlinking "Kubernetes cluster" since you already link to it in the next sentence. Having 3 links here may confuse readers looking where to go. However, 2 links may allow them to understand:
- if I have a cluster, I can follow the guide to deploy coder
- if I don't have a cluster, I can follow Coder's guide to create one

EDIT: I just noticed another link below to the specific helm install docs. I would recommend removing some since this guide is focused on "getting started." If you see value in having all of these different links, I think they should be labeled more clearly so someone understands
- ./setup/index.md: everything you need to know to set up coder is nested here
- ./setup/kubernetes/index.md: guides for different types of clusters
- ./setup/installation.md: after you have a cluster that meets the requirements, you can Install here.

I am concerned people will try to click these links in the order you provided them and may get confused.

File: getting-started.md
Comment (bpmct) [inline getting-started.md:22]: Perhaps mention "You can get a license for free [here](https://coder.com/trial)"

The product already mentions this and it's very likely trial users will be coming to this doc, but I don't want to make it seem like a paid license is required to try Coder.

File: getting-started.md
Comment (bpmct) [inline getting-started.md:46]: Perhaps we can mention that users can manually be created and invited as well?

File: getting-started.md
Comment (bpmct) [inline getting-started.md:53]: 👏🏼

File: getting-started.md
Comment (bpmct) [inline getting-started.md:57]:
```suggestion
While Coder supports a [variety of IDEs in the browser](https://coder.com/docs/coder/v1.20/workspaces/editors), such as VSCode and the
```

**Review by bpmct — CHANGES_REQUESTED** [review]

**Comment by khorne3** [comment]
I grabbed a coffee cup from [Bootstrap icons](https://icons.getbootstrap.com/); there aren't rockets, but they have angled arrows that could work as well.

**Review by khorne3 — APPROVED** [review]

**Comment by IAmHughes** [comment]
@bpmct can you re-review. With the changes from @khorne3 I think we're good to merge once approved.

File: getting-started.md
Comment (IAmHughes) [reply getting-started.md:9]: @jawnsy do you have a suggestion of what to use instead?

File: getting-started.md
Comment (IAmHughes) [reply getting-started.md]: Fixed, good catch

File: getting-started.md
Comment (khorne3) [reply getting-started.md:9]: Maybe `Coder's documentation is publically editable`?

Comment (jawnsy) [reply getting-started.md:9]: Yeah, I'd just lean away from the term "open source" and say that anyone can make suggestions/contribute edits

File: getting-started.md
Comment (jawnsy) [inline getting-started.md:26]: the `//` doesn't seem correct here? same for another instance above

Comment (bpmct) [reply getting-started.md:26]:
```suggestion
controls](./admin/access-control/index.md) with OpenID Connect (OIDC),
```

File: getting-started.md
Comment (bpmct) [reply getting-started.md:9]:
```suggestion
Coder’s [documentation is publicly editable](https://github.com/cdr/docs),
```

**Review by bpmct — APPROVED** [review]
LGTM. created suggestions for the items discussed

**Comment by khorne3** [comment]
@bpmct Thanks so much. (I'd made the fixes but forgot to push them 🤦‍♀️ )

**Comment by IAmHughes** [comment]
It looks like we're good to merge @khorne3, just wanted to make sure before I hit the button, haha

**Pull request merged**
